docs/ca/assets/app.css

```diff
@@ -0,0 +1 @@
+/home/victor/zettlr-docs/zettlr-docs/assets/app.css
```

Comment:
This is actually a hard link to the file on your own computer, but for it to work here on the CI pipeline, it needs to be relative (I think the contents of this link need to be `../../../assets/app.css`)

Reply:
Ups!

now should work